docs/develop/auth.md

```diff
@@ -149,11 +149,12 @@ func AuthHandler(ctx context.Context, token string) (auth.UID, error) {
 
 Note that for security reasons you may not want to reveal too much information about why a request did not pass your auth checks. There are many subtle security considerations when dealing with authentication and we don't have time to go into all of them here.
 
-Whenever possible we recommend using a third-party auth provider.<br/>
-See the guides for using [Firebase Authentication](/docs/how-to/firebase-auth) or [Auth0](/docs/how-to/auth0-auth) for examples of how to do that.
+Whenever possible we recommend using a third-party auth provider.
```

Comment:
Maybe worth clarifying this a bit
```suggestion
Whenever possible we recommend using a third-party auth provider instead of rolling your own authentication.
```